src/JsonLd/Serializer/ErrorNormalizer.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\JsonLd\Serializer;
+
+use ApiPlatform\State\ApiResource\Error;
+use ApiPlatform\Symfony\Validator\Exception\ValidationException as SymfonyValidationException;
+use ApiPlatform\Validator\Exception\ValidationException;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class ErrorNormalizer implements NormalizerInterface
+{
+    use HydraPrefixTrait;
+
+    public function __construct(private readonly NormalizerInterface $inner, private readonly array $defaultContext = [])
+    {
+    }
+
+    public function normalize(mixed $object, ?string $format = null, array $context = []): array|string|int|float|bool|\ArrayObject|null
+    {
+        $normalized = $this->inner->normalize($object, $format, $context);
+        $hydraPrefix = $this->getHydraPrefix($context);
```

Comment:
`$hydraPrefix = $this->getHydraPrefix($context + $this->defaultContext);`

Reply:
mhh this is quite weird isn't the default context added via the ContextBuilder anyways? Or maybe it is only through the `api_platform.serializer` option and not `serializer.default_context`?